sdc_visualization/server.py

```diff
@@ -352,7 +372,7 @@ def get_timeseries():
 
     ds.close()
     meta_vars.update({
-            "date": date.isoformat(),
+            "date": ensure_datetime(date).isoformat(), 
```

Comment:
first ensure date time and then add to this dict (just minor thing, but makes it more readable)